pkg/types/types.go

```diff
@@ -95,6 +95,7 @@ type VulnerabilityDetail struct {
 	Description      string     `json:",omitempty"`
 	PublishedDate    *time.Time `json:",omitempty"`
 	LastModifiedDate *time.Time `json:",omitempty"`
+	CPEDetails       string     `json:",omitempty"`
```

Comment:
We should have a struct for CPEDetails, not string.

Reply:
noted, I thought we want to optimize the storage here, so I tried storing as JSON string

Reply:
In any cases, we marshal VulnerabilityDetail to JSON in BoltDB, so the disk size would be almost the same. If we store `CPEDetails` as string, we need to unmarshal twice.

Reply:
Ok, pushing the change